diff --git a/packages/cli/src/services/frontend.service.ts b/packages/cli/src/services/frontend.service.ts
index 282bd3e2d2..e8472eebbe 100644
--- a/packages/cli/src/services/frontend.service.ts
+++ b/packages/cli/src/services/frontend.service.ts
@@ -332,16 +332,16 @@ export class FrontendService {
 				dismissed: [],
 			},
 			askAi: {
-				enabled: false,
+				enabled: true,
 			},
 			aiBuilder: {
-				enabled: false,
-				setup: false,
+				enabled: true,
+				setup: true,
 			},
 			aiCredits: {
-				enabled: false,
-				credits: 0,
-				setup: false,
+				enabled: true,
+				credits: -1,
+				setup: true,
 			},
 			ai: {
 				allowSendingParameterValues: true,
